server/src/main/java/org/opensearch/tasks/TaskCancellationMonitoringSettings.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tasks;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Settings related to task cancellation monitoring service.
+ */
+public class TaskCancellationMonitoringSettings {
+
+    public static final long INTERVAL_MILLIS_SETTING_DEFAULT_VALUE = 5000;
+    public static final long DURATION_MILLIS_SETTING_DEFAULT_VALUE = 10000;
+    public static final boolean IS_ENABLED_SETTING_DEFAULT_VALUE = true;
+
+    /**
+     * Defines the interval(in millis) at which task cancellation service monitors and gather stats.
+     */
+    public static final Setting<Long> INTERVAL_MILLIS_SETTING = Setting.longSetting(
+        "task_cancellation.interval_millis",
+        INTERVAL_MILLIS_SETTING_DEFAULT_VALUE,
+        1,
+        Setting.Property.NodeScope
```

Comment:
We can explicitly make the property final if we don't want user to update it

Reply:
Let move this to the service class itself.

Reply:
@jainankitk I wanted this to be updatable but will be applied during node restart. So not sure if final will give me that. I had referred few other similar settings. 
@Bukhtawar I think it is better to keep all settings explicitly here than moving to service?